tests/scalers/aws-dynamodb-streams.test.ts

```diff
@@ -0,0 +1,209 @@
+import test from 'ava'
+import * as fs from 'fs'
+import * as sh from 'shelljs'
+import * as tmp from 'tmp'
+import { CreateTableCommand, DescribeTableCommand, DeleteTableCommand, DynamoDBClient,} from '@aws-sdk/client-dynamodb'
+import { DynamoDBStreamsClient, DescribeStreamCommand } from '@aws-sdk/client-dynamodb-streams'
+import { createNamespace, sleep, waitForDeploymentReplicaCount } from './helpers'
+
+const awsRegion = 'ap-northeast-1'
+const awsAccessKey = process.env['AWS_ACCESS_KEY'];
+const awsSecretKey =  process.env['AWS_SECRET_KEY'];
+const dynamoDBStreamsNamespace = 'keda-test'
```

Comment:
This namespace is too generic, please add some prefix so it is unique for this test.

e2e tests are running in the parallel, so this might cause problems.